_docs_use-cases/use-case-consumption-delete-an-own-shared-attribute-and-notify-peer.md

```diff
@@ -0,0 +1,60 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-delete-an-own-shared-attribute-and-notify-peer
+published: true
+title: "Delete an own shared Attribute and notify peer"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate Enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA23
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: deleteOwnSharedAttributeAndNotifyPeer
+  - description:
+  - feature category: Cross-identity attribute sharing
+  - tech category: Attributes
+  - status: DONE
+  - documentation status: DONE
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex:
+  - published: default
+  - link: use-case-consumption-delete-an-own-shared-attribute-and-notify-peer
+require:
+required_by:
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete an own shared [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute), i.e. a LocalAttribute with a `shareInfo`, that is owned by you.
```

Comment:
I'd reformulate like that: "This use-case allows you to delete an own shared Attribute, which is a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) that was created as a result of sharing an Attribute that is owned by yourself with one of your peers."